components/dashboard/src/images/ideLogos.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+import goland from './golandLogo.svg';
```

Comment:
I guess we will need to host them under IDE proxy eventually to take under our contorl.